content/en/blog/2020/proxying-legacy-services-using-egress-gateways/index.md

```diff
@@ -0,0 +1,487 @@
+---
+title: Proxying legacy services using Istio egress gateways 
+subtitle: Deploy mesh egress gateways independently to allow secure connectivity between legacy and internet services
+description: Deploy multiple Istio egress gateways independently to have fine-grained control of egress communication from the mesh.
+publishdate: 2020-12-14
```

Comment:
When this PR is ready to merge, please update this date one more time. It should be set to the correct date that the PR merges.

Reply:
Alright